pkg/learning/mab_ss.go

```diff
@@ -33,12 +34,15 @@ type MABSeedScheduler struct {
 	// Corpus program array. The index in this array is the same as the
 	// index in the MAB engine.
 	corpus []*prog.Prog
+	// Reward change since last Poll.
+	rewardChange map[int]float64
```

Comment:
I really want to sync changes to totalTime and totalCov as well and restore them after fuzzer crash-and-restarts. We did that in the research with a single VM. However, that wouldn't make sense (especially the totalTime part) if different fuzzers are running on VMs with different computational powers.

Is this common in practice?

Reply:
We need to try to prototype #1541 and if it works, then it will make syncing unnecessary.

Reply:
Folding the fuzzer into the manager could definitely reduce the syncing overhead. Still, the timing measurement could be affected if there is a discrepancy between VM performances. 

Are there any solid attempts underway to making #1541 happen?

Reply:
The VMs are generally assumed to be equivalent.

> Are there any solid attempts underway to making #1541 happen?

None that I know of.